### PR TITLE
Docs: Modified Upgrade PXF when GPDB

### PR DESCRIPTION
**Pull request by mperezfuster**
Edited some of the steps when upgrading PXF as part of a GPDB upgrade.
Unfortunately I am unable to provide preview link.

### CODE REVIEW

**Review by bradfordb-vmware — APPROVED** [review]

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 1]
```diff
@@ -44,7 +48,7 @@ After you upgrade to the new version of Greenplum Database, perform the followin
     $ ssh gpadmin@<gpmaster>
     ```
 
-2. If you installed the PXF `rpm` on your Greenplum 6 hosts:
+2. If you installed the PXF `rpm` on your Greenplum 6 hosts, you will need to register it in order to continue using PXF:
```
Comment (lisakowen) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:51]: you will need to register it in order to continue using pxf -> you must register it to continue using pxf

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 2]
```diff
@@ -54,9 +58,9 @@ After you upgrade to the new version of Greenplum Database, perform the followin
 
     2. Start PXF on each segment host as described in [Starting PXF](../../pxf/5-15/cfginitstart_pxf.html#start_pxf).
 
-    3. Exit this procedure.
+    3. Skip the following steps and exit this procedure.
 
-2. Initialize PXF on each segment host as described in [Initializing PXF](../../pxf/5-15/init_pxf.html). You may choose to use your existing `$PXF_CONF` for the initialization.
+2. If you are using the PXF version included with Greenplum Database installation, initialize PXF on each segment host as described in [Initializing PXF](../../pxf/5-15/init_pxf.html). You may choose to use your existing `$PXF_CONF` for the initialization.
```
Comment (lisakowen) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:63]: what about "To continue using the PXF version included in the Greenplum ..."?  if they are on this page, they were using embedded pxf in their previous greenplum.

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 3]
```diff
@@ -6,6 +6,8 @@ If you are using PXF in your current Greenplum Database 6.x installation, you mu
 
 The PXF upgrade procedure describes how to upgrade PXF in your Greenplum Database installation. This procedure uses *PXF.from* to refer to your currently-installed PXF version and *PXF.to* to refer to the PXF version installed when you upgrade to the new version of Greenplum Database.
 
+Note: if you are planning in upgrading PXF but not as part of a Greenplum Database upgrade, refer to the [PXF Documentation](../../../pxf/5-15/upgrade_pxf_rpm.html)
```
Comment (lisakowen) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:9]: planning in -> planning on?  or maybe ... planning to upgrade PXF ...

also, i am noticing that the other links to the pxf docset include only two dir levels up (../../), not three.

**Review by lisakowen — CHANGES_REQUESTED** [review]
thanks, these changes should help reduce confusion.

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 4]
```diff
@@ -6,6 +6,8 @@ If you are using PXF in your current Greenplum Database 6.x installation, you mu
 
 The PXF upgrade procedure describes how to upgrade PXF in your Greenplum Database installation. This procedure uses *PXF.from* to refer to your currently-installed PXF version and *PXF.to* to refer to the PXF version installed when you upgrade to the new version of Greenplum Database.
 
+Note: if you are planning to upgrade PXF but not as part of a Greenplum Database upgrade, refer to the [PXF Documentation](../../pxf/5-15/upgrade_pxf_rpm.html)
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:9]: Bold the **Note:**

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 5]
```diff
@@ -54,9 +58,9 @@ After you upgrade to the new version of Greenplum Database, perform the followin
 
     2. Start PXF on each segment host as described in [Starting PXF](../../pxf/5-15/cfginitstart_pxf.html#start_pxf).
 
-    3. Exit this procedure.
+    3. Skip the following steps and exit this procedure.
 
-2. Initialize PXF on each segment host as described in [Initializing PXF](../../pxf/5-15/init_pxf.html). You may choose to use your existing `$PXF_CONF` for the initialization.
+2. To continue using the PXF version included with Greenplum Database installation, initialize PXF on each segment host as described in [Initializing PXF](../../pxf/5-15/init_pxf.html). You may choose to use your existing `$PXF_CONF` for the initialization.
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:63]: with Greenplum -> with the Greenplum

**Review by dyozie — APPROVED** [review]

**Review by ashuka24 — APPROVED** [review]

**Review by dyozie — APPROVED** [review]

**Pull request merged**